fcsparser/api.py

```diff
@@ -58,12 +59,14 @@ class FCSParser(object):
         self.channel_names_alternate holds the alternate names of the channels
     """
 
-    def __init__(self, path, read_data=True, channel_naming='$PnS'):
+    def __init__(self, path, dataset_start=0, read_data=True, channel_naming='$PnS'):
```

Comment:
dataset_start is not very convenient for a user to specify since it requires the user to extract the meta parameters and then feed the meta parameter back into the parser.

Instead of specifying an offset in terms of bytes in a binary file, how about we specify an offset in terms of dataset blocks?

For example, to get the first dataset a user would provide:

```
dataset_num = 0
```

To get the second dataset:
```
dataset_num = 1
```

and so on...